### PR TITLE
Remove post_pivot script for feature test

### PR DESCRIPTION
**Pull request by kashifest**
This PR removes post_pivot script for feature test. This script was added initially to check an operation after the pivot works or not. Currently in features test we run node_reuse after pivot so we don't need it anymore.

### CODE REVIEW

**Comment by mboukhalfa** [comment]
> /test-integration
> /test-v1a4-centos-integration
> /test-features

The test-features fails until we got the PR #766 merged

File: vm-setup/roles/v1aX_integration_test/tasks/post_pivot.yml [hunk 1]
```diff
@@ -1,71 +0,0 @@
-  - name: Define number of BMH's
-    set_fact:
-      NUMBER_OF_BMH: "{{ NUM_OF_MASTER_REPLICAS|int +  NUM_OF_WORKER_REPLICAS|int }}"
-
-  - name: Untaint all nodes.
-    shell: |
-        kubectl taint nodes --all node-role.kubernetes.io/master-
-    environment:
-      KUBECONFIG: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
-    ignore_errors: yes
```
Comment (furkatgofurov7) [inline vm-setup/roles/v1aX_integration_test/tasks/post_pivot.yml:10]: You need these lines in the beginning of `node_reuse.yml` file, specifically before the:
`   - name: Scale worker down to 0 to start testing KubeadmControlPlane node reuse test scenario.` task. 
I went inside the VM while running the feature tests and saw BMO and Ironic Pod in `pending` state in target cluster. Untainting nodes helped BMO and ironic pods to become in `running` state and pass the step where it was failing previously.

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

**Comment by furkatgofurov7** [comment]
Failed in remediation, re-triggering. 
/test-features

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Pull request merged**